src/js/core/widget/core/Spin.js

```diff
@@ -57,7 +57,7 @@
 			ROLL_DURATION = 600,
 			DELTA_Y = 100,
 			DRAG_STEP_TO_VALUE = 60,
-			NUMBER_OF_CAROUSEL_ITEMS = 11,
+			NUMBER_OF_CAROUSEL_ITEMS = 13,
```

Comment:
is this change still needed?

Reply:
It is not very necessary but extends touching area for drag/swipe.